qiskit_metal/renderers/renderer_mpl/mpl_canvas.py

```diff
@@ -12,39 +12,42 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 """MPL Canvas."""
-import logging
-import random
-import sys
+#import logging
+#import random
+#import sys
 from typing import TYPE_CHECKING, List
 
 import matplotlib
 import matplotlib as mpl
 import matplotlib.patches as patches
 import matplotlib.pyplot as plt
-import numpy as np
-import pandas as pd
+#import numpy as np
+#import pandas as pd
 from cycler import cycler
-from descartes import PolygonPatch
-from IPython.display import display
+#from descartes import PolygonPatch
+#from IPython.display import display
```

Comment:
May I suggest we delete unused imports? Commenting them out solves pylint but leaves behind unnecessary stuff.

Reply:
Agreed